blogposts/2021/creating-sourcegraph-product-design-principles.md

```diff
@@ -1,8 +1,10 @@
 ---
-title: 'When gut instinct doesn’t scale: Creating Sourcegraph’s product design principles'
+title: 'How we created Sourcegraph’s product design principles'
+externalTitle: 'Product design principles at Sourcegraph'
 author: Quinn Keast
 authorUrl: https://quinnkeast.com
 description: 'Here’s how our rapidly growing all-remote product and design teams collaborated asynchronously to define Sourcegraph’s inclusive product design principles to help us scale consistently.'
```

Comment:
```suggestion
description: 'Gut instinct doesn’t scale. Here’s how our rapidly growing all-remote product and design teams collaborated asynchronously to define Sourcegraph’s inclusive product design principles to help us scale consistently.'
```
Added this back into the description since you removed it from the title – this was before your time but people really liked that phrase so want to include it somewhere 😃